### ISSUE TITLE
Variadic parameter breaks building packages

### ISSUE THREAD

**Issue by pedestrianlove** [mono/mono#14401]
This breaks building packages, which use some harder compiler flags by default. Specifically:

```
11:00:17  marshal.c: In function 'mono_array_to_byte_byvalarray_impl':
11:00:17  marshal.c:739:3: error: format not a string literal and no format arguments [-Werror=format-security]
11:00:17     mono_error_set_argument (error, "string", gerror->message);
```

_Originally posted by @directhex in https://github.com/mono/mono/pull/14119_

**Comment by lambdageek** [comment]
Already fixed in #14385.

**Comment by pedestrianlove** [comment]
Oh my bad
